ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -154,33 +155,55 @@ import Ouroboros.Consensus.Util.IOLike
 -- For more details, see:
 -- <https://github.com/IntersectMBO/ouroboros-network/pull/1175#issuecomment-558147194>
 instance CardanoHardForkConstraints c => SerialiseHFC (CardanoEras c) where
-  encodeDiskHfcBlock (CardanoCodecConfig ccfgByron ccfgShelley ccfgAllegra ccfgMary ccfgAlonzo ccfgBabbage ccfgConway) = \case
-    -- We are backwards compatible with Byron and thus use the exact same
-    -- encoding.
-    BlockByron blockByron -> encodeDisk ccfgByron blockByron
-    -- For Shelley and later eras, we need to prepend the hard fork envelope.
-    BlockShelley blockShelley -> prependTag 2 $ encodeDisk ccfgShelley blockShelley
-    BlockAllegra blockAllegra -> prependTag 3 $ encodeDisk ccfgAllegra blockAllegra
-    BlockMary blockMary -> prependTag 4 $ encodeDisk ccfgMary blockMary
-    BlockAlonzo blockAlonzo -> prependTag 5 $ encodeDisk ccfgAlonzo blockAlonzo
-    BlockBabbage blockBabbage -> prependTag 6 $ encodeDisk ccfgBabbage blockBabbage
-    BlockConway blockConway -> prependTag 7 $ encodeDisk ccfgConway blockConway
-  decodeDiskHfcBlock (CardanoCodecConfig ccfgByron ccfgShelley ccfgAllegra ccfgMary ccfgAlonzo ccfgBabbage ccfgConway) = do
-    enforceSize "CardanoBlock" 2
-    CBOR.decodeWord >>= \case
-      0 -> fmap BlockByron <$> Byron.decodeByronBoundaryBlock epochSlots
-      1 -> fmap BlockByron <$> Byron.decodeByronRegularBlock epochSlots
-      -- We don't have to drop the first two bytes from the 'ByteString'
-      -- passed to the decoder as slicing already takes care of this.
-      2 -> fmap BlockShelley <$> decodeDisk ccfgShelley
-      3 -> fmap BlockAllegra <$> decodeDisk ccfgAllegra
-      4 -> fmap BlockMary <$> decodeDisk ccfgMary
-      5 -> fmap BlockAlonzo <$> decodeDisk ccfgAlonzo
-      6 -> fmap BlockBabbage <$> decodeDisk ccfgBabbage
-      7 -> fmap BlockConway <$> decodeDisk ccfgConway
-      t -> cborError $ DecoderErrorUnknownTag "CardanoBlock" (fromIntegral t)
-   where
-    epochSlots = Byron.getByronEpochSlots ccfgByron
+  encodeDiskHfcBlock
+    ( CardanoCodecConfig
+        ccfgByron
+        ccfgShelley
+        ccfgAllegra
+        ccfgMary
+        ccfgAlonzo
+        ccfgBabbage
+        ccfgConway
+        ccfgDijkstra
+      ) = \case
+      -- We are backwards compatible with Byron and thus use the exact same
+      -- encoding.
```

Comment:
Minor: does our current formatter allow us to align this code block? It'd make it easier to understand the structure.

Reply:
I've disabled the formatter and aligned the code